www/widgets/DemoExampleMaker.js

```diff
@@ -296,7 +317,7 @@ class DemoExampleMaker extends Widget {
 
     this.$('[name="dem-s-title"]').value = step.title
     this.$('[name="dem-s-focus"]').value = step.focus
-    this.$('[name="dem-s-text"]').value = step.text
+    this._text.code = step.text
```

Comment:
when i press "add new step" i get an error

i think the issue is with this line. should be:
```js
this._text.code = step.text || 'in this step...'
```